src/Constants/constants.tsx

```diff
@@ -0,0 +1,55 @@
+import { AvatarInfoType } from "../Types/types";
+export const fetchApi = (url: string) => {
+    return fetch(url);
+};
+
+const activePort: string = "4001"
+
+export const getAllUsersUrl: string = `http://localhost:${activePort}/getAllUsers`
+
+export const getUserMessagesUrl = (userId: string) : string => {
+    return `http://localhost:${activePort}/getUserMessages/${userId}`
+}
+
+export const addUserMessageUrl =(userId: string) : string => {
+    return `http://localhost:${activePort}/addMessage/${userId}`
+}
```

Comment:
Keep these URL helpers with their respective hooks. If we require same query at multiple places we can just reuse the query hook itself.